src/net/account/Account.cairo

```diff
@@ -0,0 +1,178 @@
+%lang starknet
```

Comment:
I think we can replace this with a script that fetches contract from the source and generates a compiled json file.